frontend/src/component/LogEntries.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { lensProp, omit, over, uniq } from 'ramda'
+import Query from 'http/Query'
+import LogEntry from 'component/LogEntry'
+import { lang } from 'util/preferences'
+
+const getTranslatedName = organizationName => organizationName[lang] || ''
+const nameLens = lensProp('organizationName')
+
+const LogEntries = () => (
+  <Query url='logs'>
+    {({ data, error, pending }) => {
+      if (error) return <div>{'Oops!'}</div>
+      if (pending) return <div>{'Wait!'}</div>
```

Comment:
Should we have localized "Oops!" and "Wait!" texts?

Reply:
Oh yes indeed, these were supposed to be just placeholders 🤦‍♂️

Reply:
(the `pending` should actually be an ajax animation or something like that)